tests/NServiceKit.Text.Tests/Utils/DateTimeSerializerTests.cs

```diff
@@ -160,7 +160,7 @@ public void UtcDateTime_Is_Deserialized_As_Kind_Utc()
 			var serialized = JsonSerializer.SerializeToString(utcNow);
 			//Deserializing UTC?
 			var deserialized = JsonSerializer.DeserializeFromString<DateTime>(serialized);
-			Assert.That(deserialized.Kind, Is.EqualTo(DateTimeKind.Utc)); //fails -> is DateTimeKind.Local
+			Assert.That(deserialized.Kind, Is.EqualTo(DateTimeKind.Utc)); 
```

Comment:
The comment said this test was failing but I checked it again and it passed fine, so I removed the Ignore tag.